src/commands/unstage.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
             const tokenInfo = await provider.getTokenInfo(argv.currency);
             
             const amount = utils.parseAmount(argv.amount, tokenInfo.decimals);
+            if (amount.eq(0))
```

Comment:
Would it be reasonable to reject negative values?

Reply:
Good catch. Will update accordingly.